### ISSUE TITLE
Release v2.26.0 throws error: "aws_flow_log.this is empty tuple"

### ISSUE THREAD

**Issue by surminus** [terraform-aws-modules/terraform-aws-vpc#405]
I know it only got released very recently (13 minutes ago as far as I can tell :eyes:) but running Terraform against v2.26.0 caused this error:

```
  on .terraform/modules/vpc/terraform-aws-modules-terraform-aws-vpc-2ac8bf5/outputs.tf line 1008, in output "vpc_flow_log_id":
1008:   value       = aws_flow_log.this[0].id
    |----------------
    | aws_flow_log.this is empty tuple

The given key does not identify an element in this collection value.
```

Pinned it back to 2.25.0 for the time-being since I'm not configuring flow logs, which is the [only change in this release](https://github.com/terraform-aws-modules/terraform-aws-vpc/blob/master/CHANGELOG.md#v2260---2020-03-11).

EDIT: worth noting my Terraform version, which _is_ slightly out of date:

```
$ terraform --version     
Terraform v0.12.20
+ provider.aws v2.52.0
```

**Comment by antonbabenko** [comment]
Thanks, Laura, for the report. I will take a look at this right now.

**Comment by antonbabenko** [comment]
Fixed in 8554bfd.

[v2.27.0](https://github.com/terraform-aws-modules/terraform-aws-vpc/releases/v2.27.0) has been just released.

**Comment by poolski** [comment]
Damn, fast work! 👍
